new-pipeline/src/dashboards.js

```diff
@@ -1073,16 +1073,18 @@ function buildDictionaryURL(metric, channel, description) {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = buildDictionaryURL(metric, channel, description);
-  var descr = metric;
+  var descr;
+
   if (description && (description.length > 0)) {
-    descr = description;
+  	descr = description;
```

Comment:
A minor thing: this looks like you used a tab instead of two spaces?
Lets change this two spaces, that is our standard indentation here.

Reply:
I am Sorry. I'll change it.